Docs/reference/requests_and_deploys.md

```diff
@@ -0,0 +1,73 @@
+# Requests and Deploys
+
+Singularity uses the concepts of a `Request` and `Deploy` to keep track of changes to a particular task or set of related tasks.
+
+A `Request` can be thought of as the high level information for a single project or deployable item. For example, the name and owner's email for a single web service, or the name and cron schedule for a single scheduled job.
+
+A `Deploy` can then be thought of as the specific configuration or version of the running code for that deployable item.
+
+To illustrate the differences between and usages of these two concepts, we will walk through an example using Singularity to run a single web service.
+
+## The `TestService` Example
+
+
+### Creating a `Request`
+You have a new web service called `TestService` that you want to run via Singularity. The first thing you need to do is create a `Request` for `TestService`. To create this request, you would `POST` json over http to the Singularity API or create a request via the new request page in the Singularity UI. Example json:
+
+```json
+{
+    "id": "TestService",
+    "requestType": "SERVICE",
+    "owners": ["me@test.com"],
+    "instances": 1,
+}
+```
+
+This `Request` now holds high level information about your web service. Singularity knows it is a long running `SERVICE` named `TestService` and that you want to run `1` instance of this service.
+
+### Creating a `Deploy`
+
+Now you want to actually start a task for `TestService`. To do this, you need to create a `Deploy` for the `TestService` `Request`. This deploy will let Singularity know all of the information necessary to actually build and launch a task. This information includes things like the command to run, environment variables to set, the location of any artifacts to download, or the resources that should be allocated to a task. You would create this deploy by `POST`ing json to the Singularity API's deploy endpoint, or creating a new deploy in the Singularity UI. Example json:
```

Comment:
"now you actually want to start a task" may not make sense to new users -- "Now you want TestService to actually run" may make more sense